sim_city.py

```diff
@@ -0,0 +1,125 @@
+import random
+import math
+import logbook
+
+BASE_CITY_INITIAL_TAX = 1000
+logger = logbook.Logger('city')
+
+
+class City:
+
+    def __init__(self):
+        self.neighborhoods = []
+        self.base_city_tax = BASE_CITY_INITIAL_TAX
+
+    def how_much_money(self):
+        neighborhoods_tax = sum([neighborhood.calculate_tax() for neighborhood in self.neighborhoods])
+        return self.base_city_tax + neighborhoods_tax
+
+    def build_a_neighborhood(self, neighborhood_name):
+        if self._find_neighborhood(neighborhood_name) is not None:
+            logger.info(f'A neighborhood with the name {neighborhood_name} already exists!')
```

Comment:
I would raise an exception in this case (and other cases when the user tries to do an illegal action).


In addition, if the neighborhoods names are unique and you need to frequently check if the name already exists, maybe it will be more convenient to store the neighborhoods in a dictionary?
Then `_find_neighborhood` will be very simple :)